src/dials/command_line/scale.py

```diff
@@ -97,6 +97,10 @@
       .type = int
       .help = "Number of bins to use for calculating and plotting merging stats."
       .expert_level = 1
+    additional_stats = False
+      .type = bool
+      .help = "Show additional merging statistics"
```

Comment:
As above, it would be nice for the user to know _what_ additional statistics will be shown.